tasks/undercloud.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: get package facts
+  package_facts:
+    manager: "auto"
+
+- name: Is the undercloud containerized?
+  set_fact:
+    containerized_undercloud: "{{ ansible_facts.packages['puppet-keystone'][0].version is version('13', '>=') }}"
```

Comment:
Does this work upstream too?

Reply:
Not sure actually ... Do you have another test instead?  Look for the keystone container?